src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/Queues/CosmosQueueClient.cs

```diff
@@ -453,7 +454,7 @@ public async Task CompleteJobAsync(JobInfo jobInfo, bool requestCancellationOnFa
                 .WithParameter("@groupId", groupId.ToString())
                 .WithParameter("@queueType", queueType);
 
-            var response = await ExecuteQueryAsync(sqlQuerySpec, 100, cancellationToken);
+            var response = await ExecuteQueryAsync(sqlQuerySpec, 100, queueType, cancellationToken);
```

Comment:
QQ - does this limit the number of responses to only 100? For very large export operations, could this cause an issue?

I saw an issue when building the export response with this code as well - it will only fetch the export file links for the first one hundred group jobs it finds.

```c#
var response = await ExecuteQueryAsync(sqlQuerySpec, 100, cancellationToken);
```eems to limit the number of responses to only 100. For large export operations, could this cause an issue? Would this new "safety" logic be moot for larger exports?

I think I saw an issue when building the export response with this code as well - it will only fetch the export file links for the first 100 group jobs it finds.

```c#
var response = await ExecuteQueryAsync(sqlQuerySpec, 100, cancellationToken);
```

Reply:
Sorry I didn't reply here, but no specific reason to limit. I think 100 felt high but not infinite